src/platform/lumin-runtime/elements/builders/ui-node-builder.js

```diff
@@ -21,6 +21,9 @@ export class UiNodeBuilder extends TransformBuilder {
     constructor(){
         super();
 
+        this._propertyDescriptors['height'] = new PrimitiveTypeProperty('height', undefined, undefined, 'number');
```

Comment:
I'm not sure if this is the right approach. Nikolay, will this interfere with how we are currently handling width/height properties (manually instead of via _propertyDescriptors)? I added these to cause the typescript generator to include them but they are probably not valid for every UiNode subclass.

Reply:
You are right, it will interfere. The base code in ElementBuilder will try to validate them while they are used only for some of the components and very often only for creating the component.

Reply:
Lets remove `width` and `height` for now. We might re-work that once we migrate to adapters instead of builders.

Reply:
Done. I think I'll just add a hook to ElementBuilder to allow custom typescript code to be injected into the Prop Interface -- that will allow me to add width and height where needed as well as missing callbacks like onClick.